src/templates/users/userprofile_detail.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% load static %}
+
+{% block title %}
+  User:
+  {{ object.dispaly_name }}
```

Comment:
I'm assuming you meant "object.display_name", not "object.dispaly_name"

Reply:
Good catch! Fixed the typo